src/stories/20_Features/Visual_Effects.html

```diff
@@ -0,0 +1,106 @@
+<!--
```

Comment:
Sorry for not being clear enough. I meant a new category "Visual Effects" and the sample would be named Ken Burns. You can do this by renaming the file to:

```
src/stories/30_Visual_Effects/Ken_Burns.html
```

Creating a new directory (30_Visual_Effects) will create a new category. You can copy and adjust the index.json file from one of the other directories (it contains the category description).